pepper-apis/studybuilder-cli/studies/osteo/study-workflows.conf

```diff
@@ -171,6 +171,51 @@
         },
       ]
     },
+    {
+      "from": {
+        "type": "ACTIVITY"
+        "activityCode": "CONSENT",
+      },
+      "to": [
+        {
+          "type": "ACTIVITY",
+          "activityCode": "CONSENT_ADDENDUM",
+          "expression": """user.studies["CMI-OSTEO"].forms["CONSENT_ADDENDUM"].hasInstance()"""
+        },
+        {
+          "type": "ACTIVITY",
+          "activityCode": "RELEASE_SELF",
+          "expression": """user.studies["CMI-OSTEO"].forms["RELEASE_SELF"].hasInstance()"""
+        }
+      ]
+    },
+    {
+      "from": {
+        "type": "ACTIVITY"
```

Comment:
I believe this flow TO can be added to above one at the end and get rid of this one.
Same for below one too but doesn't hurt to have diff one